package.json

```diff
@@ -28,7 +28,9 @@
     "server": "node src/backend/web/server",
     "hint": "hint http://localhost:3000",
     "webhint": "cross-env PORT=3000 npm-run-all -r -p server hint",
-    "debug-server": "nodemon server.js"
+    "debug-server": "nodemon server.js",
+    "lighthouse-index": "lighthouse http://localhost:3000 --output-path=./index-report.report.html --view",
```

Comment:
We'll need to pre-launch the web server on port 3000 for your script to work.  We did something similar for `webhint` above.  You need to do:

```
"lighthouse-index": "lighthouse http://localhost:3000 --output-path=./lighthouse-report.html --view",
"lighthouse": "cross-env PORT=3000 npm-run-all -r -p server lighthouse-index",
```

Also, I renamed the output path for the report.  We won't have a separate login.html page soon, so there's no point testing it.